docs/api/trigger_run_parameters.md

```diff
@@ -19,34 +20,98 @@ So the job configurations are typically not a suitable model for end users to sp
+* create specific job configurations for parameters; e.g., the users can specify a set of FossID parameters, which are then added to the corresponding options in the scanner configuration
+
+For example, a validation script could use the parameters to configure the scanner and evaluator jobs.
+
+The following input:
+
+```json
+{
+  "jobConfigs": {
+    "parameters": {
+      "applicationCategory": "web",
+      "useFossId": true,
+      "fossIdInstance": "QM",
+      "fossIdDeltaScans": 7
+    }
+  }
+}
+```
+
+Could be transformed to these job configurations:
+
+```json
+{
+  "resolvedJobConfigs": {
+    "scanner": {
+      "scanners": [
+        "FossId"
+      ],
+      "config": {
+        "FossId": {
+          "options": {
+            "serverUrl": "https://fossid-qm.example.org",
+            "deltaScans": true,
+            "deltaScanLimit": 7
+          },
+          "secrets": {
+            "user": "fossIdUser_QM",
+            "apikey": "fossIdApiKey_QM"
+          }
+        }
+      }
+    },
+    "evaluator": {
+      "ruleSet": "web.rules.kts",
+      "licenseClassificationsFile": "web.license-classifications.yml"
+    }
+  }
+}
+```
 
 Here the request passed to Orchestrator contains a number of simple key-value pairs. The validation script reads this input and performs the following transformations:
 
 * It detects that FossID should be used. Therefore, it makes sure that the output contains a `ScannerJobConfiguration` with options that include a map for FossID-related parameters.
 * Users specify the FossID instance to be used as an enumeration constant - _QM_ in this case. The script can check whether this is an allowed value. If so, it adds the data of the selected instance to the FossID configuration - in this case the URL and the credentials. _Note:_ The credentials are represented as names of secrets that need to be looked up later, not as the actual secret values.
-* Further parameters related to FossID, e.g. the project name or the number of delta scans to keep, are also added to the FossID options in the scanner configuration.
-* The _applicationCategory_ parameter is also checked for correctness. In this example, the category determines the Evaluator configuration to be used - the rule set and the license classifications. The validation script therefore creates an `EvaluatorJobConfiguration` and populates it accordingly.
+* Further parameters related to FossID, e.g., the number of delta scans to keep, are also added to the FossID options in the scanner configuration.
+* The _applicationCategory_ parameter is also checked for correctness. In this example, the category determines the Evaluator configuration to be used: the rule set and the license classifications. The validation script therefore creates an `EvaluatorJobConfiguration` and populates it accordingly.
```

Comment:
I find this sentence before and after hard to read. Maybe "to be used, i.e. the" would be better?